src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberMethodSymbol.cs

```diff
@@ -1613,35 +1613,44 @@ internal override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, r
         {
             base.AddSynthesizedAttributes(moduleBuilder, ref attributes);
 
-            if (this.IsAsync || this.IsIterator)
+            bool isAsync = this.IsAsync;
+            bool isIterator = this.IsIterator;
```

Comment:
Is there a reason that these should be locals? Curious if it's just style or you're protecting against mutations that happen below.

Reply:
I made those locals because they are used multiple times, but there is no fundamental or other reason.

---
In reply to: [221787737](https://github.com/dotnet/roslyn/pull/30105#discussion_r221787737) [](ancestors = 221787737)